ui/src/literature/containers/DetailPageContainer/DetailPageContainer.jsx

```diff
@@ -123,7 +123,7 @@ function DetailPage({ authors, record, referencesCount, supervisors }) {
                   <>
                     {citationCount != null && (
                       <IncomingLiteratureReferencesLinkAction
-                        recordId={controlNumber}
+                        linkQuery={`refersto:recid:${controlNumber}`}
```

Comment:
```suggestion
                        linkQuery={getPapersQueryString(controlNumber)}
```

I'd move in `ui/src/literature/utils.js``

```js
export function getPapersQueryString(recordId) {
  return `refersto:recid:${recordId}`;
}
```